### PR TITLE
Modified Component: gucharmap enabled 64bit build

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

File: components/gucharmap/Makefile [hunk 1]
```diff
@@ -49,9 +49,9 @@ CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
 COMPONENT_INSTALL_ENV+=	GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
```
Comment (pyhalov) [inline components/gucharmap/Makefile:51]: Please, if you touch Makefile, increment COMPONENT_REVISION, move it to appropriate category, remove BUILD_PKG_DEPENDS and depend.mk, add REQUIRED_PACKAGES

File: components/gucharmap/gucharmap.p5m [hunk 2]
```diff
@@ -242,3 +242,12 @@ file path=usr/share/omf/gucharmap/gucharmap-vi.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_CN.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_HK.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_TW.omf
+
```
Comment (pyhalov) [inline components/gucharmap/gucharmap.p5m]: please, use gmake sample-manifest and add necessary actions in corresponding place

**Comment by Toasterson** [comment]
What would be the appropriate category?
components/desktop/gnome2 ?

**Comment by pyhalov** [comment]
I think so

**Comment by Toasterson** [comment]
And Fixed

File: components/desktop/gnome2/gucharmap/Makefile [hunk 3]
```diff
@@ -27,6 +28,7 @@ COMPONENT_FMRI= desktop/character-map/gucharmap
 COMPONENT_CLASSIFICATION= Applications/System Utilities
 COMPONENT_LICENSE= GPLv3
 COMPONENT_LICENSE_FILE= COPYING
+COMPONENT_REVISION = 1
```
Comment (pyhalov) [inline components/desktop/gnome2/gucharmap/Makefile:32]: Let's do it consistent, please, place it after COMPONENT_VERSION

File: components/desktop/gnome2/gucharmap/Makefile [hunk 4]
```diff
@@ -10,6 +10,7 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov
+# Copyright (c) 2016 Till Wegmueller
 #
 
 include ../../make-rules/shared-macros.mk
```
Comment (pyhalov) [inline components/desktop/gnome2/gucharmap/Makefile]: You have to fix include path

**Comment by pyhalov** [comment]
change commit message to "gucharmap: enable 64bit build"  and let's be done with it

**Pull request merged**